### DISCUSSION TITLE
source_view_link not working

### DISCUSSION THREAD

**Discussion by trim21** [pradyunsg/furo#802] [Potential Issue] [answered]
I'm trying to use this option but find it's missing.

https://github.com/pradyunsg/furo/blob/main/src/furo/theme/furo/theme.conf

sphinx also raise a warning:

unsupported theme option 'source_view_link' given

**Comment by 99phari** [comment]
I have the same issue.

According to the documentation [Changing buttons on page > With arbitrary URLs](https://pradyunsg.me/furo/customisation/top-of-page-buttons/#with-arbitrary-urls) this should be working since version `2024.05.06`.

I am on that version and get the same warning.

Here a minimal `conf.py` to reproduce:

```py
html_theme = 'furo'

html_theme_options = {
    'source_edit_link': 'https://www.google.com',  # for trials lets just go to google
    'source_view_link': 'https://www.google.com',  # this causes the described warning
}

```

The "edit this page" button works as it should (redirects to google in this example).
The "view this page" button simply opens the `filename.rst.txt`.

**Reply by q-wertz** [reply]
Same issue here.

Versions:

```bash
$ pip list | grep furo
furo                          2024.5.6
$ pip list | grep sphinx
sphinx-autobuild              2024.4.16
sphinx-autodoc-typehints      2.2.2
sphinx-basic-ng               1.0.0b2
sphinx-copybutton             0.5.2
sphinx_design                 0.6.0
sphinxcontrib-applehelp       1.0.8
sphinxcontrib-bibtex          2.6.2
sphinxcontrib-devhelp         1.0.6
sphinxcontrib-drawio          0.0.17
sphinxcontrib-htmlhelp        2.0.5
sphinxcontrib-jsmath          1.0.1
sphinxcontrib-qthelp          1.0.7
sphinxcontrib-serializinghtml 1.1.10
sphinxcontrib-tikz            0.4.20
```

**Comment by pradyunsg** [comment, marked as answer]
🙈

I know what's going on. I'll file a fix after my work day.

**Reply by pradyunsg** [reply]
Fixed in 2024.07.18
